### PR TITLE
Added Gas Refinery card

### PR DESCRIPTION
**Pull request by Lakshmii3**

### CODE REVIEW

File: backend/game/newcards/discard_cards/victory_point_manipulation_cards/test_gas_refinery.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from game.newcards.discard_cards.victory_point_manipulation_cards.gas_refinery import \
+    GasRefinery
+
+
+def test_gas_refinery_costs_6_energy():
+    assert GasRefinery().cost == 6
+
+
+def test_gas_refinery_adds_2_victory_points(player, five_players):
```
Comment (borisv13) [inline backend/game/newcards/discard_cards/victory_point_manipulation_cards/test_gas_refinery.py]: adds 2 victory points to whom?

File: backend/game/newcards/discard_cards/victory_point_manipulation_cards/test_gas_refinery.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    GasRefinery().immediate_effect(player, five_players)
+    assert player.victory_points == 2
+
+
+def test_gas_refinery_subtracts_3_health(player, five_players):
```
Comment (borisv13) [inline backend/game/newcards/discard_cards/victory_point_manipulation_cards/test_gas_refinery.py]: subtracts 3 health from whom?

File: backend/game/newcards/discard_cards/victory_point_manipulation_cards/test_gas_refinery.py (hunk 1 above)
Comment (Lakshmii3) [reply backend/game/newcards/discard_cards/victory_point_manipulation_cards/test_gas_refinery.py]: Modified test case name to "test_gas_refinery_adds_2_victory_points_to_current_player"

File: backend/game/newcards/discard_cards/victory_point_manipulation_cards/test_gas_refinery.py (hunk 2 above)
Comment (Lakshmii3) [reply backend/game/newcards/discard_cards/victory_point_manipulation_cards/test_gas_refinery.py]: Modified test case name to "test_gas_refinery_subtracts_3_health_from_other_players"

**Pull request merged**